modules/internal/ChapelDebugPrint.chpl

```diff
@@ -26,7 +26,7 @@
  */
 
 module ChapelDebugPrint {
-  private use ChapelStandard, SysCTypes;
+  private use ChapelStandard, SysCTypes, IO;
```

Comment:
Is this necessary?

Reply:
I needed this when I was using `chpl_debug_writeln`. I think we need it, but I will remove it from this PR to make it more self-contained and not to open a can of worms because of the module init order.